src/openforms/forms/migrations/0104_formstatistics.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 3.2.23 on 2024-01-19 11:42
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("forms", "0103_fix_component_problems"),
```

Comment:
can you see if you can make this migration depend on `0091_v230_to_v250` instead? that's the squashed migration including everything up to (and including) version 2.5.0